rust/perspective-python/pyproject.toml

```diff
@@ -11,19 +11,27 @@
 #  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 
 [build-system]
-requires = [
-    "setuptools",
-    "wheel",
-    "numpy>=1.13.1,<2",
-    "jupyter-packaging==0.12.3",
+requires = ["maturin>=1.4,<2.0"]
+build-backend = "maturin"
+
+[project]
+name = "perspective-python"
+requires-python = ">=3.8"
+dynamic = ["version"]
+classifiers = [
+    "Programming Language :: Rust",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
 ]
-build-backend = "jupyter_packaging.build_api"
 
-[tool.black]
-line-length = 200
-target-version = ['py37', 'py38']
-include = '\.pyi?$'
+[project.optional-dependencies]
+dev = ["pytest>=7.4.3", "ruff>=0.4.6"]
+
+[tool.maturin]
```

Comment:
we should just make sure to double check that the JS/JSON assets get bundled here now that we moved off the old packaging style, if Maturin isn't doing the correct thing by default and recursively including the package tree, we can easily explicitly include the files: https://www.maturin.rs/config#maturin-options

Reply:
#2658